openapi.yaml

```diff
@@ -23,6 +24,8 @@ tags:
     description: Environment operations
   - name: ACL
     description: ACL (Access Control List) operations on topics and consumer groups
+  - name: SchemaRequest
+    description: SchemaRequest operations
```

Comment:
Nitpick: I perceive these tags as the "entity" the operations are dealing with. If this assumption is correct, should this be "Schema operation"?

Reply:
That's something I've on my list of questions :D Because based on properties and everything, the operation and model are specifically concerned with **requests** for/of Schemas 🤔  

(also, super minor detail, but we import `Schema` from "zod" into some places, so using `SchemaRequest` avoids having to rename imports. But that's really minor)

Reply:
Haha I noticed that as well:
```yaml
  - name: User
    description: User API endpoints
```

Maybe we should use "API endpoint" instead of "operations", then it makes more sense to use the entity name?